azure-pipelines-v3.yml

```diff
@@ -17,10 +17,10 @@ jobs:
 
   # Install .NET Core sdk
   - task: UseDotNet@2
-    displayName: 'Install .NET Core sdk 2.2.300'
+    displayName: 'Install .NET Core sdk 3.1.100'
```

Comment:
consider extracting it to a variable: https://github.com/dotnet/docfx/pull/5397/files#diff-bee4d79cd349ddb0cb2b4b7d5a8bf3d3R10